dotnet/src/SemanticKernel/Kernel.cs

```diff
@@ -112,21 +91,21 @@ public ISKFunction RegisterSemanticFunction(string skillName, string functionNam
         foreach (KeyValuePair<string, ISKFunction> f in skill)
         {
             f.Value.SetDefaultSkillCollection(this.Skills);
-            this._skillCollection.AddNativeFunction(f.Value);
+            this._skillCollection.AddFunction(f.Value);
         }
 
         return skill;
     }
 
     /// <inheritdoc/>
-    public ISKFunction RegisterCustomFunction(string skillName, ISKFunction customFunction)
+    public ISKFunction ImportFunction(string skillName, ISKFunction customFunction)
```

Comment:
Not sure about the terms `Import` / `Add` we clearly are using different ones for ISkillCollection and IKernel. IMO would use one term for all I like `Add`.

Reply:
see other comment